Userland/Libraries/LibWebView/ProcessManager.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2024, Andrew Kaster <akaster@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/EventLoop.h>
+#include <LibCore/System.h>
+#include <LibWebView/ProcessManager.h>
+
+namespace WebView {
+
+static sig_atomic_t s_received_sigchld = 0;
+
+ProcessType process_type_from_name(StringView name)
+{
+    if (name == "WebContent")
+        return ProcessType::WebContent;
+    if (name == "WebWorker")
+        return ProcessType::WebWorker;
+    if (name == "SQLServer")
+        return ProcessType::SQLServer;
+    if (name == "RequestServer")
+        return ProcessType::RequestServer;
+    if (name == "ImageDecoder")
+        return ProcessType::ImageDecoder;
+
+    dbgln("Unknown process type: '{}'", name);
+    VERIFY_NOT_REACHED();
+}
+
+StringView process_name_from_type(ProcessType type)
+{
+    switch (type) {
+    case ProcessType::Chrome:
+        return "Chrome"sv;
+    case ProcessType::WebContent:
+        return "WebContent"sv;
+    case ProcessType::WebWorker:
+        return "WebWorker"sv;
+    case ProcessType::SQLServer:
+        return "SQLServer"sv;
+    case ProcessType::RequestServer:
+        return "RequestServer"sv;
+    case ProcessType::ImageDecoder:
+        return "ImageDecoder"sv;
+    default:
+        VERIFY_NOT_REACHED();
```

Comment:
I would put this assertion outside of the `switch`, so we get a compile error when a new type is added but not included here, rather than a runtime failure